crates/bevy_ecs/src/schedule/state.rs

```diff
@@ -279,7 +279,7 @@ impl<T: Component + Clone + Eq> State<T> {
     }
 
     pub fn inactives(&self) -> &[T] {
-        &self.stack[0..self.stack.len() - 2]
+        self.stack.split_last().map(|(_last, rest)| rest).unwrap()
```

Comment:
```suggestion
        self.stack.split_last().map(|(_, rest)| rest).unwrap()
```
Is there a reason to name the unused value? I feel this is more idiomatic.